aurora/src/test/java/com/market/banica/aurora/config/ChannelManagerTest.java

```diff
@@ -0,0 +1,142 @@
+package com.market.banica.aurora.config;
+
+import com.market.banica.aurora.model.ChannelProperty;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+@ExtendWith(MockitoExtension.class)
+class ChannelManagerTest {
+    private static final String CHANNEL_KEY = "market/europe";
+    private static final String CHANNEL_PREFIX_MARKET = "market";
+    private static final String CHANNEL_PREFIX_WEATHER = "weather";
+    private static final String CHANNEL_PREFIX_ASTERISK = "*";
+
+    private static final String HOST = "localhost";
+    private static final int PORT = 1010;
+    private static final int EDITED_PORT = 1011;
+    private static ChannelProperty channelProperty;
+
+
+    @Mock
+    private ManagedChannel managedChannel = ManagedChannelBuilder
```

Comment:
why is this mocked and initialized at the same time. Shouldn't it be mocked only ?